mira/dkg/askemo/askemo.json

```diff
@@ -240,6 +248,20 @@
     "physical_min": 0.0,
     "suggested_data_type": "float",
     "suggested_unit": "1 / day",
+    "synonyms": [
+      {
+        "type": "referenced_by_latex",
+        "value": "\\delta_1"
+      },
+      {
+        "type": "referenced_by_latex",
+        "value": "\\delta_2"
+      },
+      {
+        "type": "referenced_by_latex",
+        "value": "\\delta_3"
```

Comment:
I don't think the numbering should be used here - those are arbitrary and not generally useful. We can just use `delta` as a single relevant symbol.

Reply:
Done in 11f68a3